### PR TITLE
docs: Added network threat manager page

### PR DESCRIPTION
**Pull request by pierantoniomerlino**
> **Note**: We are using the Conventional Commits convention for our pull request titles. Please take a look at the [PR title format document](https://github.com/eclipse/kura/blob/develop/CONTRIBUTING.md#submitting-the-changes) for the supported [types](https://github.com/eclipse/kura/blob/develop/CONTRIBUTING.md#type) and [scopes](https://github.com/eclipse/kura/blob/develop/CONTRIBUTING.md#scope).

This PR adds a page in the configuration section that describes the Network Threat Manager feature for both IPv4 and IPv6.

### CODE REVIEW

File: docs/gateway-configuration/network-threat-manager.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# Network Threat Manager
+
+Eclipse Kura provides a set of features to detect and prevent network attacks. The Security section in the Gateway Administration Console shows the Network Threat Manager tab where is it possible to activate these functions.
+
+!!! Warning
+    The Network Threat Manager tab is not available for the No Network version of Eclipse Kura.
```
Comment (mattdibi) [inline docs/gateway-configuration/network-threat-manager.md]: For clarity I would link the explaination of what we mean by "No Network version"

```suggestion
    The Network Threat Manager tab is not available for the [No Network version of Eclipse Kura](../getting-started/install-kura/#installer-types).
```

File: docs/gateway-configuration/network-threat-manager.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+iptables -A prerouting-kura -p tcp --tcp-flags ALL NONE -j DROP
+iptables -A prerouting-kura -p tcp --tcp-flags ALL FIN,PSH,URG -j DROP
+iptables -A prerouting-kura -p tcp --tcp-flags ALL SYN,FIN,PSH,URG -j DROP
+iptables -A prerouting-kura -p tcp --tcp-flags ALL SYN,RST,ACK,FIN,URG -j DROP
+iptables -A prerouting-kura -p icmp -j DROP
+iptables -A prerouting-kura -f -j DROP
+```
+
+To further filter the incoming TCP fragmented packets, specific system configuration files are configured.
+The _flooding.protection.enabled_ property is used to enable the feature.
```
Comment (mattdibi) [inline docs/gateway-configuration/network-threat-manager.md]: Is \` a better way to format this?

```suggestion
The `flooding.protection.enabled` property is used to enable the feature.
```

Feel free to ignore this comment if you don't agree

**Review by mattdibi — CHANGES_REQUESTED** [review]

File: docs/gateway-configuration/network-threat-manager.md (hunk 2 above)
Comment (pierantoniomerlino) [reply docs/gateway-configuration/network-threat-manager.md]: Indeed we have not an agreement on with typographical enphasis use for the property names. In some pages we have bold, while in other italics or code style.

Comment (pierantoniomerlino) [reply docs/gateway-configuration/network-threat-manager.md]: Looking at the other pages, it seems that the property names are in bold.

File: docs/gateway-configuration/network-threat-manager.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
+# Network Threat Manager
+
+Eclipse Kura provides a set of features to detect and prevent network attacks. The Security section in the Gateway Administration Console shows the Network Threat Manager tab where is it possible to activate these functions.
+
+!!! Warning
+    The Network Threat Manager tab is not available for the [No Network version of Eclipse Kura](../getting-started/install-kura/#installer-types).
```
Comment (mattdibi) [inline docs/gateway-configuration/network-threat-manager.md]:
```suggestion
    The Network Threat Manager tab is not available for the [No Network version of Eclipse Kura](../../getting-started/install-kura/#installer-types).
```

**Review by mattdibi — APPROVED** [review]

**Pull request merged**